catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet

```diff
@@ -0,0 +1,3859 @@
+        description: 'Unique visit number to the NEON site',
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'CITATION',
+        description: 'Data citation. See ' +
+        '[NEON Data Policies and Citation Guidelines]' +
+        '(https://www.neonscience.org/data-samples/data-policies-citation)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'DOI',
+        description: 'Digital Object Identifier. NEON data that have been released are assigned a DOI.',
+        type: ee_const.var_type.string,
+      },      
+      {
+        name: 'FLIGHT_YEAR',
+        description: 'Year the data were collected',
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'NEON_DOMAIN',
+        description:
+          'NEON eco-climatic domain code, "D01" to "D20". See ' +
+          '[NEON Field Sites and Domains]' +
+          '(https://www.neonscience.org/field-sites/about-field-sites)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_SITE',
+        description: 'NEON four-digit site code. See ' +
+        '[NEON Field Sites](https://www.neonscience.org/field-sites/)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_ID',
+        description: 'NEON data product identification code. Always set to: "DP3.30006.001"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_URL',
+        description: 'NEON data product url. Always set to: ' +
+        '[https://data.neonscience.org/data-products/DP3.30010.001]' +
+        '(https://data.neonscience.org/data-products/DP3.30010.001)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'PROVISIONAL_RELEASED',
+        description: 'Whether the data are Provisional or Released. See ' +
+        '[https://www.neonscience.org/data-samples/data-management/data-revisions-releases]' +
+        '(https://www.neonscience.org/data-samples/data-management/data-revisions-releases)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'RELEASE_YEAR',
+        description: 'If data are released, the year of the NEON Release Tag.',
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'SCALE_FACTOR',
+        description: 'Reflectance scale factor used',
+        type: ee_const.var_type.double,
+      },
+      {
+        name: 'SENSOR_ID',
+        description: 'ID of NEON Imaging Spectrometer (NIS). The Global Airborne Observatory (GAO) sensor has also been used to collect the NEON D20 site: "NIS1", "NIS2", "NIS3", "GAO"',
+        type: ee_const.var_type.double,
+      },
+    ],
+    gsd: [
+        1.0
+    ],
+    platform: [
+      'NEON',
+    ],
+    instruments: [
+      'GAO',
+      'NIS1', 
+      'NIS2',
+      'NIS3',
+    ],
+    'eo:bands': [
+     {
+      name: "B001",
+      description: "VNIR Band B001",
+      'gee:units': 'unitless',
```

Comment:
We don't typically say "unitless" for such bands... though Hyperion's units are  'W/m^2 SR&mu;m',

Have you see reflectance units be specified in other places?